docs/library/models/organizing.md

```diff
@@ -67,16 +67,16 @@ service = ModelLibrary(
 
 ### Abstract models
 
-It is possible to define models that inherits from an abstract model in order to share common behavior.
+It is possible to define models that inherits from an abstract model in order to share common behavior. It only requires to not set CONFIGURATIONS dict for those models to be ignored from the configuration steps.
```

Comment:
Shouldn't we add an error message if someone does it?

Reply:
If someone define abstract model ?
No, that is actually what we want to support. Like we did until 0.0.18